src/device.rs

```diff
@@ -101,26 +118,95 @@ impl Device {
         Ok(Self::from_raw(udev, ptr))
     }
 
-    /// Creates a device for a given major/minor number
+    /// Creates a rust udev `Device` for a given UNIX device "special file" type and number. Uses
+    /// an existing [`Udev`] instance rather than creating one automatically.
```

Comment:
I think this might be a copy-paste error? The `_with_context` variants use an existing instance.